RuleDocumentation/AvoidUsingConvertToSecureStringWithPlainText.md

```diff
@@ -4,29 +4,33 @@
 
 ## Description
 
-The use of the `AsPlainText` parameter with the `ConvertTo-SecureString` command can expose secure information.
+The use of the `AsPlainText` parameter with the `ConvertTo-SecureString` command can expose secure
+information.
 
 ## How
 
 Use a standard encrypted variable to perform any SecureString conversions.
 
 ## Recommendations
 
-If you do need an ability to retrieve the password from somewhere without prompting the user, consider using Windows Credential Store as used in the BetterCredentials module ( https://www.powershellgallery.com/packages/BetterCredentials/4.4/Content/BetterCredentials.psm1). If that does not work, consider using Azure KeyVault (https://azure.microsoft.com/en-us/services/key-vault/) or AWS KMS (https://aws.amazon.com/kms/).
+If you do need an ability to retrieve the password from somewhere without prompting the user,
+consider using Windows Credential Store as used in the
```

Comment:
Should point to SecretStore instead of Windows Credential Store

Reply:
Fixed